### PR TITLE
Add filter during remote commands to machines only in maintenance mode

### PR DESCRIPTION
**Pull request by mseidl**
## What does this PR change?
Filters machines not in maintenance mode when using remote commands. 

## GUI diff

- [ ] **DONE**

## Documentation
- No documentation needed:  

- [ ] **DONE**

## Test coverage
- No tests: 

- [ ] **DONE**

## Links

No specific issue/bug, part of bigger card
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by ncounter** [comment]
> Looks good so far from the code POV.
> 
> I'd also add information feedback to the user actually.
> A dynamic sentence would be the best option, telling the user there were systems in maintenance mode and they were filtered out by the remote command action.
> But as an easy first solution I'd add a static sentence in the webUI page stating that If maintenance windows are used and some of the systems are in maintenance mode, they are excluded from the execution and the result of the remote command applied.
> 

After the discussion that happened offline, we figured out the behavior is the opposite, so it is fine to not add any message. The only systems they will show up are the one in maintenance mode, so there is no need to highlight it in words.

File: java/code/src/com/suse/manager/webui/websocket/RemoteMinionCommands.java [hunk 1]
```diff
@@ -133,8 +135,10 @@ public void onMessage(Session session, String messageBody) {
             int timeOut = 600; // 10 minutes
 
             if (msg.isPreview()) {
+
                 List<String> allVisibleMinions = MinionServerFactory
                         .lookupVisibleToUser(webSession.getUser())
+                        .filter(m -> !mm.isSystemInMaintenanceMode(m))
```
Comment (mcalmer) [inline java/code/src/com/suse/manager/webui/websocket/RemoteMinionCommands.java:141]: I think the ! (not) is wrong here. When a system is *in* maintenance mode, we can run remote commands.

Comment (ncounter) [reply java/code/src/com/suse/manager/webui/websocket/RemoteMinionCommands.java:141]: Agreed, we discussed offline and I think Martin is about to fix it.

**Comment by mcalmer** [comment]
@mseidl - Backport to 4.1?

**Pull request merged**